src/urls/package-url-handler.ts

```diff
@@ -86,7 +86,7 @@ export class PackageUrlHandler implements UrlHandler {
    * package.
    */
   getOriginalPackageNameForUrl(url: OriginalDocumentUrl): string {
-    if (url.startsWith('../')) {
+    if (url.startsWith('bower_components/')) {
```

Comment:
this change seems to be going from detecting component-style cross-package references, to app-style - what's this for?

Reply:
My understanding is that a `PackageUrlHandler` is only used when converting in project mode, so dependencies of the main package will always appear under `bower_components/`. @FredKSchott, please correct me if I'm wrong.

Reply:
Following from that, I *think* that dependencies in workspace mode are all converted using a single `WorkspaceUrlHandler`, rather than individual `PackageUrlHandler`s.

Reply:
- this has nothing to do with element vs. application. It has to do with the URL format the analyzer gives us (pre 3.0 it gave us `../`, post 3.0 it gives us 'bower_components/')